threshsign/include/threshsign/bls/relic/Library.h

```diff
@@ -69,14 +69,12 @@ class LibraryInitializer {
 
 class Library {
  public:
-  static Library* GetPtr() {
+  static const Library& Get() {
     // C++11 guarantees this is thread-safe (but all threads will share this variable!)
-    static Library* lib = new Library();
+    static const Library lib;
```

Comment:
js moving the Library object from the heap to the stack done on purpose?

Reply:
Yes, it's a small object, its size is known at compile time, and there is no real reason to allocate it on the heap at first.